source/install/packstack.html.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Packstack quickstart
+title: Packstack
 category: documentation
 authors: apevec, dneary, garrett, jasonbrooks, jlibosva, mmagr, pixelbeat, pmyers,
   rbowen, gbraad, cbrown2
 wiki_category: Documentation
-wiki_title: Quickstart
+wiki_title: Packstack
```

Comment:
I think it was to keep trace of the origin, but it's old now and nobody cares, so I'm also in favor of removing them. Also writers keep misunderstanding and adding wiki headers for new content so…